src/Nest/Requests.NoNamespace.cs

```diff
@@ -618,13 +618,6 @@ protected DeleteByQueryRequest(): base()
 			set => Q("analyze_wildcard", value);
 		}
 
-		///<summary>The analyzer to use for the query string</summary>

```

Comment:
breaking change?

Reply:
yes, looks like it was removed from the spec: 

- 7.3: https://github.com/elastic/elasticsearch/blob/7.3/rest-api-spec/src/main/resources/rest-api-spec/api/delete_by_query.json
- 7.4: https://github.com/elastic/elasticsearch/blob/7.4/rest-api-spec/src/main/resources/rest-api-spec/api/delete_by_query.json